src/core/core.scale.js

```diff
@@ -1214,11 +1214,11 @@ var Scale = Element.extend({
 			switch (scaleLabelAlign) {
 			case 'start':
 				scaleLabelX = me.left;
-				textAlign = 'left';
+				textAlign = me.reverse ? 'right' : 'left';
```

Comment:
This should be the same `me.options.ticks.reverse`

So add a `var reverse = me.options.ticks.reverse;` and use that in all these.

Reply:
Looks like I forgot to update this one